src/util/instanceBulkActions.tsx

```diff
@@ -0,0 +1,98 @@
+import {
+  LxdInstance,
+  LxdInstanceAction,
+  LxdInstanceStatus,
+} from "types/instance";
+import { InstanceBulkAction } from "api/instances";
+
+// map desired actions to pairs of instance status and performed action
+// statuses missing will be ignored
+const actionsToPerform: Partial<
+  Record<
+    LxdInstanceAction,
+    Partial<Record<LxdInstanceStatus, LxdInstanceAction>>
+  >
+> = {
+  start: {
+    Frozen: "unfreeze",
+    Stopped: "start",
+  },
+  restart: {
+    Running: "restart",
+  },
+  freeze: {
+    Running: "freeze",
+  },
+  stop: {
```

Comment:
I think we should enable actions for the "freezing" state. Currently, we can restart and stop a freezing instance from the detail page, but not from here. This can be useful when instances get stuck in the freezing state, like it often happens on the demo server. The freezing instance is even escaping the "ignored" counter currently:

![Screenshot from 2023-04-19 18:02:21](https://user-images.githubusercontent.com/56583786/233135735-953815dd-2f8f-4751-9a17-81d890d05d95.png)

```suggestion
    Freezing: "restart",
    Running: "restart",
  },
  freeze: {
    Running: "freeze",
  },
  stop: {
    Freezing: "stop",
```

Reply:
We can't restart a frozen instance. The api responds with `Instance is not running`.

As for the screenshot, I can't reproduce that any more.

Reply:
I wasn't talking about _frozen_ instances, I was talking about _freezing_ instances. They can be restarted and stopped, no?

![image](https://user-images.githubusercontent.com/56583786/233321018-db6d6134-6f58-408b-a2dc-6369b5feb7a6.png)

As for reproducing, it's very simple. The instance stuck in the freezing state is still there: https://lxd-ui-293.demos.haus/ui/default/instances

Just select it and another one (I used the frozen one right below it for the example above) and try clicking on the allowed actions (start or stop), and you'll reproduce the issue easily. It's something that should be addressed.

Reply:
Ah right, we have also the `Starting` and `Freezing` states. I added the later one to restart and stop and the starting to stop as well.